### PR TITLE
2 sign insign up page

### LINKED ISSUE #2: Sign In/Sign Up Page
**Task title**
Sign In/Sign Up Page

**Task description**
Create a Sign In/Sign Up Page and its Components

**Subtasks**

1. Input Fields
 - Sign In
- [x] Email
- [x] Password 
 - Sign Up
- [x] Email
- [x] UserName
- [x] Password
- [x] Confirm Password
2. Button
- [x] Sign In
- [x] Sign Up
3. Links
- [x] Forget Password?
4. Social Sign-In Options
- [x] Google (Firebase)
5. Messages
- [x] Validation
- [x] Error
- [x] Success
6. Firebase
- [x] Auth
- [x] Firestore

**Link to the component on Figma**
[Figma component](https://www.figma.com/file/sDlEio3FtEwhvkQQyYJUZS/share-and-care?type=design&node-id=68-4057&mode=design&t=EfzNZY3MzXDepRr0-4)

### PR DESCRIPTION
**Pull request by mouloud247**
1. Build resetPassword component

2. Build signOut component

3. Link signUp with Firestore

4. Add hover effect to the Buttons with some styling for the Forms

**Screenshots**

image
image
image
image

# Related Issue

- Resolve #2

### CODE REVIEW

File: src/components/Auth/resetPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    <div>
+                        <button
+                            type='submit'
+                            className='w-full bg-teal-500 text-white py-2 px-4 rounded hover:shadow-lg transform hover:scale-105'
+                        >
+                            Reset Password
+                        </button>
+                    </div>
+                </form>
+                <div>{message && <p>{message}</p>}</div>
```
Comment (ikoworld) [inline src/components/Auth/resetPassword.jsx:52]: can you explain this line of code please !

**Review by walidbelharazem — COMMENTED** [review]
thanks for your hard work, the components are working correctly just want to mention if you can avoid using absolute cause it gonna be difficult to manipulate the responsiveness.

File: src/components/Auth/resetPassword.jsx
Comment (ikoworld) [inline src/components/Auth/resetPassword.jsx:1]: code clear well organized you have met all the requirement well done :)

**Review by ikoworld — APPROVED** [review]

**Review by walidbelharazem — APPROVED** [review]

File: src/components/Auth/resetPassword.jsx (hunk 1 above)
Comment (mouloud247) [reply src/components/Auth/resetPassword.jsx:52]: This is a conditional rendering, if message exists, it will be rendered.

Comment (ikoworld) [reply src/components/Auth/resetPassword.jsx:52]: understood

**Pull request merged**